.run/silo--api.run.xml

```diff
@@ -1,10 +1,15 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="silo --api" type="CMakeRunConfiguration" factoryName="Application" PROGRAM_PARAMS="--api --dataDirectory output/" REDIRECT_INPUT="false" ELEVATE="false" USE_EXTERNAL_CONSOLE="false" EMULATE_TERMINAL="false" WORKING_DIR="file://.." PASS_PARENT_ENVS_2="true" PROJECT_NAME="SILO" TARGET_NAME="siloApi" CONFIG_NAME="Debug" RUN_TARGET_PROJECT_NAME="SILO" RUN_TARGET_NAME="siloApi">
-    <envs>
-      <env name="SPDLOG_LEVEL" value="debug" />
-    </envs>
-    <method v="2">
-      <option name="com.jetbrains.cidr.execution.CidrBuildBeforeRunTaskProvider$BuildBeforeRunTask" enabled="true" />
-    </method>
-  </configuration>
+    <configuration default="false" name="silo --api" type="CMakeRunConfiguration" factoryName="Application"
```

Comment:
This does not change the functionality, does it? My Clion just revertet this back to the previous format. So I would not add this change here.

Reply:
Huh, my Clion changed it actually to what I have there now. There was an update to clang tidy, do you have that as well?